OrbitGl/DataView.cpp

```diff
@@ -102,19 +100,19 @@ void DataView::ExportCSV(const std::string& a_FileName) {
 
 void DataView::CopySelection(const std::vector<int>& selection) {
   std::string clipboard;
-  size_t numColumns = GetColumns().size();
-  for (size_t i = 0; i < numColumns; ++i) {
+  size_t num_columns = GetColumns().size();
+  for (size_t i = 0; i < num_columns; ++i) {
     clipboard += GetColumns()[i].header;
-    if (i < numColumns - 1) clipboard += ", ";
+    if (i < num_columns - 1) clipboard += ", ";
```

Comment:
Nit: I know this hasn't actually changed, but since you're refactoring it: I think that would benefit from absl::StrJoin: `clipboard = absl::StrJoin(GetColumns(), ", ", [](std::string* out, const auto& column) { out.append(column.header); });`

Reply:
The last commit does this, please have a look if this is ok/better as there are now two nested `absl::StrJoin`s. In particular I don't like that I need to create an array with the column indices.

Reply:
Ahh. I see your point. I'm sorry, I didn't see that GetValue needs an instance. I also don't particularly like that. And there is no good solution, at least not without a lazy-range-library like cppitertools or range-v3. So feel free to revert that.

Reply:
Yeah, I think I'll revert it also to keep the parallelism with `ExportCSV` above.